types/moment-locales-webpack-plugin/tsconfig.json

```diff
@@ -1,24 +1,28 @@
 {
     "compilerOptions": {
-        "baseUrl": "../",
-        "forceConsistentCasingInFileNames": true,
-        "lib": ["es6"],
         "module": "commonjs",
-        "noEmit": true,
+        "lib": [
+            "es6"
+        ],
         "noImplicitAny": true,
         "noImplicitThis": true,
-        "strictFunctionTypes": true,
         "strictNullChecks": true,
-        "typeRoots": ["../"],
+        "strictFunctionTypes": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
         "paths": {
             "webpack": [
-                "webpack/v4"
+                "./node_modules/webpack"
             ],
             "tapable": [
-                "tapable/v1"
+                "./node_modules/tapable"
             ]
```

Comment:
Do you still need these entries if you're leveraging `package.json` dependencies for these imports?

Reply:
I think so; I did the same as other packages did. e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56391

Reply:
Can we try without and see if CI passes? I just looked through a bunch of other packages with dependencies and I’m pretty sure this shouldn’t be needed.

Reply:
Ah wait, the issue is that there’s still a `types/webpack` and `types/tapable` folder, so this _is_ needed in order not to use those.